rectools/models/ease.py

```diff
@@ -93,6 +112,13 @@ def _fit(self, dataset: Dataset) -> None:  # type: ignore
         self.weight = np.array(gram_matrix_inv / (-np.diag(gram_matrix_inv)))
         np.fill_diagonal(self.weight, 0.0)
 
+    @property
+    def _recommend_use_gpu_ranking(self) -> bool:
+        use_gpu = HAS_CUDA
+        if self.recommend_use_gpu_ranking is False:
+            use_gpu = False
```

Comment:
```suggestion
        use_gpu = self.recommend_use_gpu_ranking is not False and HAS_CUDA
```

Reply:
fixed and removed from property because we call it only in one method